code/modules/mob/living/carbon/xenomorph/castes/defender/castedatum_defender.dm

```diff
@@ -144,3 +144,151 @@
+
+	tier = XENO_TIER_ONE
+	upgrade = XENO_UPGRADE_BASETYPE
+	wound_type = "defender" //used to match appropriate wound overlays
+
+	// *** Melee Attacks *** //
+	melee_damage = 17
+
+	// *** Speed *** //
+	speed = -0.3
+
+	// *** Plasma *** //
+	plasma_max = 150
+	plasma_gain = 10
+
+	// *** Health *** //
+	max_health = 260
+
+	// *** Evolution *** //
+	evolution_threshold = 80
+	upgrade_threshold = 60
+
+	evolves_to = list(/mob/living/carbon/xenomorph/warrior)
+
+	// *** Flags *** //
+	caste_flags = CASTE_CAN_BE_QUEEN_HEALED|CASTE_EVOLUTION_ALLOWED|CASTE_CAN_BE_GIVEN_PLASMA|CASTE_CAN_BE_LEADER|CAN_BECOME_KING
+
+	// *** Defense *** //
+	soft_armor = list("melee" = 30, "bullet" = 30, "laser" = 25, "energy" = 20, "bomb" = XENO_BOMB_RESIST_2, "bio" = 20, "rad" = 20, "fire" = 30, "acid" = 20)
+
+	// *** Ranged Attack *** //
+	charge_type = CHARGE_TYPE_LARGE
+
+	// *** Defender Abilities *** //
+	crest_defense_armor = 22
+	crest_defense_slowdown = 0.8
+	fortify_armor = 50
+
+	actions = list(
+		/datum/action/xeno_action/xeno_resting,
+		/datum/action/xeno_action/activable/psydrain,
+		/datum/action/xeno_action/activable/headbite,
+		/datum/action/xeno_action/activable/devour,
+		/datum/action/xeno_action/toggle_crest_defense,
+		/datum/action/xeno_action/fortify,
+		/datum/action/xeno_action/activable/forward_charge,
+		/datum/action/xeno_action/tail_sweep,
+		/datum/action/xeno_action/activable/regenerate_skin,
+	)
+
+
+/datum/xeno_caste/defoonder/young
+	upgrade_name = "Young"
+
+	upgrade = XENO_UPGRADE_ZERO
+
+/datum/xeno_caste/defoonder/mature
+	upgrade_name = "Mature"
+	caste_desc = "He attac. He smac."
+	upgrade = XENO_UPGRADE_ONE
+
+	// *** Speed *** //
+	speed = -0.4
+
+	// *** Plasma *** //
+	plasma_max = 175
+	plasma_gain = 13
+
+	// *** Health *** //
+	max_health = 280
+
+	// *** Evolution *** //
+	upgrade_threshold = 180
+
+	// *** Defense *** //
+	soft_armor = list("melee" = 33, "bullet" = 33, "laser" = 28, "energy" = 25, "bomb" = XENO_BOMB_RESIST_2, "bio" = 25, "rad" = 25, "fire" = 33, "acid" = 25)
+
+	// *** Defender Abilities *** //
+	crest_defense_armor = 26
+	crest_defense_slowdown = 0.8
+	fortify_armor = 52
+
+/datum/xeno_caste/defoonder/elder
+	upgrade_name = "Elder"
+	caste_desc = "He attac. He smac. He snac."
+	upgrade = XENO_UPGRADE_TWO
+
+	// *** Melee Attacks *** //
+	melee_damage = 21
+
+	// *** Speed *** //
+	speed = -0.5
+
+	// *** Plasma *** //
+	plasma_max = 190
+	plasma_gain = 14
+
+	// *** Health *** //
+	max_health = 300
+
+	// *** Evolution *** //
+	upgrade_threshold = 420
+
+	// *** Defense *** //
+	soft_armor = list("melee" = 35, "bullet" = 35, "laser" = 30, "energy" = 28, "bomb" = XENO_BOMB_RESIST_2, "bio" = 28, "rad" = 28, "fire" = 35, "acid" = 28)
+
+	// *** Defender Abilities *** //
+	crest_defense_armor = 30
+	crest_defense_slowdown = 0.8
+	fortify_armor = 55
+
+/datum/xeno_caste/defoonder/ancient
+	upgrade_name = "Ancient"
+	caste_desc = "He attac. He smac. He snac. But most importantly, He PROTEC!"
```

Comment:
```suggestion
	caste_desc = "He deffen. He protecc. He fuwutify. But most importantly, He THICC!"
```

Reply:
"But most importantly, He THICC!"
What in the f...?

Reply:
Excellent